mantle/cmd/kola/options.go

```diff
@@ -210,8 +210,6 @@ func syncOptionsImpl(useCosa bool) error {
 			kola.QEMUOptions.Firmware = "uefi"
 		} else if kola.Options.CosaBuildArch == "x86_64" && kola.QEMUOptions.Native4k {
 			kola.QEMUOptions.Firmware = "uefi"
-		} else {
-			kola.QEMUOptions.Firmware = "bios"
 		}
```

Comment:
Conversely, if the QEMU code treats the empty string as `uefi` on platforms where UEFI is supported, the remaining part of this conditional serves no purpose.

Reply:
So yes...you're right *but* this also serves as a marker that this functionality actually *requires* uefi.

For example, one thing I can imagine us doing in the future is auto-generating a test matrix that e.g. runs some tests in BIOS, some in UEFI.

Hmm, perhaps what we want is `kola.QEMUOptions.RequireUEFI` - and hence it could *also* run in `uefi-secure` (which is a 3rd option in this matrix, that arguably instead should actually be the default...).